drivers/gles3/shader_compiler_gles3.cpp

```diff
@@ -79,6 +79,12 @@ static int _get_datatype_size(SL::DataType p_type) {
 		case SL::TYPE_SAMPLER2D: return 16;
 		case SL::TYPE_ISAMPLER2D: return 16;
 		case SL::TYPE_USAMPLER2D: return 16;
+		case SL::TYPE_SAMPLER2DARRAY: return 16;
+		case SL::TYPE_ISAMPLER2DARRAY: return 16;
+		case SL::TYPE_USAMPLER2DARRAY: return 16;
+		case SL::TYPE_SAMPLER3D: return 16;
+		case SL::TYPE_ISAMPLER3D: return 16;
+		case SL::TYPE_USAMPLER3D: return 16;
```

Comment:
```
drivers/gles3/shader_compiler_gles3.cpp:55:9: warning: enumeration value 'TYPE_ISAMPLER2DARRAY' not handled in switch [-Wswitch]
drivers/gles3/shader_compiler_gles3.cpp:55:9: warning: enumeration value 'TYPE_ISAMPLER3D' not handled in switch [-Wswitch]
drivers/gles3/shader_compiler_gles3.cpp:55:9: warning: enumeration value 'TYPE_SAMPLER2DARRAY' not handled in switch [-Wswitch]
drivers/gles3/shader_compiler_gles3.cpp:55:9: warning: enumeration value 'TYPE_SAMPLER3D' not handled in switch [-Wswitch]
drivers/gles3/shader_compiler_gles3.cpp:55:9: warning: enumeration value 'TYPE_USAMPLER2DARRAY' not handled in switch [-Wswitch]
drivers/gles3/shader_compiler_gles3.cpp:55:9: warning: enumeration value 'TYPE_USAMPLER3D' not handled in switch [-Wswitch]
```
@karroffel I did not check what the expected datatype size should be, please advise.